packages/yoga/src/Result/native/Result.jsx

```diff
@@ -36,6 +38,25 @@ const Content = styled.View`
   }}
 `;
 
+const TitleAndBadgeContainer = styled(Box)`
+  flex-direction: row;
+  align-items: center;
+`;
+
+const ExclusiveBadge = () => (
+  <Box
+    ml="xxxsmall"
+    bg="neon"
+    justifyContent="center"
+    alignItems="center"
+    borderRadius="circle"
+    w="small"
+    h="small"
+  >
+    <Icon as={WellhubIcon} size={10.67} fill="text.primary" />
+  </Box>
+);
+
```

Comment:
I think it's time to extract these styles to a separate file, what do you think?

Reply:
@eduardobreno Great!
Done.